cmd/mz/compress.go

```
@@ -172,9 +221,23 @@ Options:`)
 			dstFile, err := os.OpenFile(*out, os.O_CREATE|os.O_WRONLY|os.O_TRUNC, os.ModePerm)
 			exitErr(err)
 			defer dstFile.Close()
-			bw := bufio.NewWriterSize(dstFile, int(sz*2))
+			bw := bufio.NewWriterSize(dstFile, blockSizeInt*2)
 			defer bw.Flush()
```

golangci-lint (2.12.2)

[error] Error return value of `bw.Flush` is not checked

(errcheck)